cypress/integration/examples/albumCards.spec.js

```diff
@@ -0,0 +1,49 @@
+context('Album Cards', () => {
+  beforeEach(() => {
+    cy
+    .fixture('freakOut.js').as('freakOut')
```

Comment:
Think we can remove this line as the fixture is being defined on line 7

Reply:
done